kumascript/macros/js_property_attributes.ejs

```diff
@@ -9,104 +9,103 @@
 // 
 // {{js_property_attributes(1,1,0)}}
 
-var header;
-var writable_name;
-var enumerable_name;
-var configurable_name;
-var yes;
-var no;
+const text = mdn.localStringMap({
+  "en-US": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
+  },
+  "es": {
+    header: `Atributos de la propiedad <code>${env.title}</code>`,
+    writableName: "Sobrescribir",
+    enumerableName: "Numerable",
+    configurableName: "Configurable",
+    yes: "Sí",
+    no: "No"
+  },
+  "fr": {
+    header: `Attributs de <code>${env.title}</code>`,
+    writableName: "Écrivable",
+    enumerableName: "Énumérable",
+    configurableName: "Configurable",
+    yes: "Oui",
+    no: "Non"
+  },
+  "ja": {
+    header: `<code>${env.title}</code> のプロパティ属性`,
+    writableName: "書込可能",
+    enumerableName: "列挙可能",
+    configurableName: "設定可能",
+    yes: "可",
+    no: "不可"
+  },
+  "ko": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
+  },
+  "pt-BR": {
+    header: `Property attributes of <code>${env.title}</code>`,
+    writableName: "Writable",
+    enumerableName: "Enumerable",
+    configurableName: "Configurable",
+    yes: "yes",
+    no: "no"
```

Comment:
ping @mdn/yari-content-pt-br for suggestions

Reply:
@mdn/yari-content-pt-br Feel free to open a follow-up PR. 🙏